docs/installing_deis/vagrant.rst

```diff
@@ -31,18 +31,20 @@ The ``Vagrantfile`` requires the plugin `vagrant-triggers`_. To install the plug
     website.
 
 
-Generate SSH Key
-----------------
-
-.. include:: ../_includes/_generate-ssh-key.rst
-
-
 Generate a New Discovery URL
 ----------------------------
 
 .. include:: ../_includes/_generate-discovery-url.rst
 
 
+SSH Key
```

Comment:
On second thought "Generate" with the note will be better than only "SSH Key" or with another word. It will be equal than any other doc and the ``note`` alerts that it's not a "Key generation".